ml-algorithms/src/main/java/org/opensearch/ml/engine/function_calling/BedrockMessage.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.function_calling;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class BedrockMessage implements LLMMessage {
+
+    private String role;
+    private List<Object> content = new ArrayList<>();
+
+    BedrockMessage() {
+        this("user");
+    }
+
+    BedrockMessage(String role) {
+        this(role, null);
+    }
+
+    BedrockMessage(String role, List<Object> content) {
+        this.role = role;
+        if (content != null) {
```

Comment:
This null check doesn't add any value addition, can be simply assign it: 
this.content = content;